package.json

```diff
@@ -4,8 +4,10 @@
   "main": "index.js",
   "scripts": {
     "test": "ava",
-    "lint": "eslint ./",
-    "semantic-release": "semantic-release"
+    "lint": "tslint ./",
```

Comment:
Use eslint for linting

https://medium.com/palantir/tslint-in-2019-1a144c2317a9